tools/st2ctl

```diff
@@ -122,12 +122,18 @@ function register_content() {
   echo "Registering content..."
 
   if [ ! ${1} ]; then
```

Comment:
Yeah this makes sense. Also, the --help passing into register-content was just weird.